substrate/client/db/src/utils.rs

```diff
@@ -527,7 +563,7 @@ where
 	let block_gap = db
 		.get(COLUMN_META, meta_keys::BLOCK_GAP)
 		.and_then(|d| Decode::decode(&mut d.as_slice()).ok());
-	debug!(target: "db", "block_gap={:?}", block_gap);
+	info!(target: "db", "block_gap={:?}", block_gap);
```

Comment:
Same as above comment - is this conversion from debug to info deliberate or should we revert it?